raiden-ts/src/raiden.ts

```diff
@@ -990,7 +991,7 @@ export class Raiden {
   public async getUDCCapacity(): Promise<BigNumber> {
     const balance = await this.deps.latest$.pipe(pluck('udcBalance'), first(isntNil)).toPromise();
     const blockNumber = this.state.blockNumber;
-    const owedAmount = Object.values(this.state.path.iou)
+    const owedAmount = Object.values(this.state.iou)
```

Comment:
Was `iou` the only thing we stored in the path state? that is why it was moved to the root state?

Reply:
Yes, exactly. This PR is about simplifying state schema, which were a little cluttered without need. See the linked issue description for arguments.